src/Settings/App.SettingsGroup.json

```diff
@@ -235,12 +214,6 @@
     "type":             "string",
     "default":     ""
 },
-{
-    "name":             "defaultFirmwareType",
```

Comment:
Yeah, this is screwed up. There are three settings that are basically doing the similar things: 
* AppSettings.offlineEditingFirmwareClass - This is used by PlanView as the firmware type for plan creation when you are not connected to a vehicle. It is initially set by the user in the first run dialogs. You can also change from the Plan View Mission Start item when offline.
* AppSettings.defaultFirmwareType - This is no longer used. I think it may have been leftover from the change to the offline* settings.
* FirmwareUpgradeSettings.defaultFirmwareType - This remembers the last selection the user used for firmware flashing.

The idea is to remember whether the user is a PX4 or an ArduPilot user. I think the solution is to make all of these just be/reference AppSettings.offlineEditingFirmwareClass.